src/csharp/Grpc.Core/NativeDeps.Mac.csproj.include

```diff
@@ -0,0 +1,17 @@
+<Project>
+  <ItemGroup>
+    <!-- We are relying on run_tests.py to build grpc_csharp_ext with the right bitness
+    and we copy it as both x86 (needed by net45) and x64 (needed by netcoreapp1.0) as we don't
+    know which one will be needed to run the tests. -->
+    <Content Include="..\..\..\libs\$(NativeDependenciesConfigurationUnix)\libgrpc_csharp_ext.dylib">
+      <Link>libgrpc_csharp_ext.x86.dylib</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
I think PreserveNewest is the action that should be used by default (and it's also what you get when you do `dotnet migrate`). It might look a bit cryptic, but it's a well known thing in the msbuild world. It copies the file to the output directory if it has changed (and that's generally what you want).